apps/web/src/lib/mock/organizationOverview.ts

```diff
@@ -7,7 +12,7 @@ interface MockData {
   editor: string;
 }
 
-const mockdata: MockData[] = [
+const mockdata1: ProjectMockData[] = [
```

Comment:
Pick a better name for this, and also as in `organizations.ts` are those types strictly necessary?

```suggestion
const projectMockData = [...] as const;
```

Reply:
name changed, the types can stay as long as we may need them since they are there only for our convenience.